src/core/dev_api/include/augru_cell.hpp

```diff
@@ -0,0 +1,65 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "openvino/op/op.hpp"
+#include "openvino/op/util/activation_functions.hpp"
+#include "openvino/op/util/rnn_cell_base.hpp"
+
+namespace ov {
```

Comment:
maybe it's better to use ov::op::internal namespace?

Reply:
done